substrate/frame/collective/src/lib.rs

```diff
@@ -229,6 +361,23 @@ pub mod pallet {
 		/// The maximum weight of a dispatch call that can be proposed and executed.
 		#[pallet::constant]
 		type MaxProposalWeight: Get<Weight>;
+
+		/// Origin from which any proposal may be disapproved.
```

Comment:
Can you briefly explain what it means for a proposal to be "disapproved"?

Reply:
updated